<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(5, 16, 99);
border: 10px double rgb(163, 168, 19);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="millionaires">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>5</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Modern No. 20</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(163, 168, 19);
}</string>
    </property>
    <property name="text">
     <string>Millionaires</string>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(223, 228, 79);
}</string>
    </property>
    <property name="text">
     <string>Witamy!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerALabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>270</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(223, 228, 79);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerCLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>270</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(223, 228, 79);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerBLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>160</y>
      <width>270</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(223, 228, 79);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="answerDLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>260</y>
      <width>270</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(223, 228, 79);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="answerAButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(156, 131, 8);
border-top: 1px solid rgb(255, 255, 255);
border-left: 1px solid rgb(255, 255, 255);
border-right: 1px solid black;
border-bottom: 1px solid black;
}
:hover{
background-color: rgb(181, 153, 9);
}
:pressed{
background-color: rgb(206, 175, 10);
border-top: 1px solid black;
border-left: 1px solid black;
border-right: 1px solid rgb(255, 255, 255);
border-bottom: 1px solid rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerBButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(156, 131, 8);
border-top: 1px solid rgb(255, 255, 255);
border-left: 1px solid rgb(255, 255, 255);
border-right: 1px solid black;
border-bottom: 1px solid black;
}
:hover{
background-color: rgb(181, 153, 9);
}
:pressed{
background-color: rgb(206, 175, 10);
border-top: 1px solid black;
border-left: 1px solid black;
border-right: 1px solid rgb(255, 255, 255);
border-bottom: 1px solid rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerCButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(156, 131, 8);
border-top: 1px solid rgb(255, 255, 255);
border-left: 1px solid rgb(255, 255, 255);
border-right: 1px solid black;
border-bottom: 1px solid black;
}
:hover{
background-color: rgb(181, 153, 9);
}
:pressed{
background-color: rgb(206, 175, 10);
border-top: 1px solid black;
border-left: 1px solid black;
border-right: 1px solid rgb(255, 255, 255);
border-bottom: 1px solid rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="answerDButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(156, 131, 8);
border-top: 1px solid rgb(255, 255, 255);
border-left: 1px solid rgb(255, 255, 255);
border-right: 1px solid black;
border-bottom: 1px solid black;
}
:hover{
background-color: rgb(181, 153, 9);
}
:pressed{
background-color: rgb(206, 175, 10);
border-top: 1px solid black;
border-left: 1px solid black;
border-right: 1px solid rgb(255, 255, 255);
border-bottom: 1px solid rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>20</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 0, 0);
color: rgb(0, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>30</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(156, 131, 8);
border-top: 1px solid rgb(255, 255, 255);
border-left: 1px solid rgb(255, 255, 255);
border-right: 1px solid black;
border-bottom: 1px solid black;
}
:hover{
background-color: rgb(181, 153, 9);
}
:pressed{
background-color: rgb(206, 175, 10);
border-top: 1px solid black;
border-left: 1px solid black;
border-right: 1px solid rgb(255, 255, 255);
border-bottom: 1px solid rgb(255, 255, 255);
}</string>
    </property>
    <property name="text">
     <string>Graj</string>
    </property>
   </widget>
   <widget class="QLabel" name="pointsLabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: rgb(223, 228, 79);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
